mgbdis.py

```diff
@@ -1144,12 +1220,51 @@ def write_makefile(self):
 
         f.close()
 
-
+class CharacterMap():
+    def __init__(self, name, path):
+        self.name = name        
+        self.path = path
+        self.character_map = {}
+        self.max_length = 1
+    def read_number(number_string):
+        number_string = number_string.strip()
+        if number_string.startswith("$"):
+            return int("0x"+number_string[1:], 16)
```

Comment:
This should handle RGBASM's digit separators:
```suggestion
            return int(number_string[1:].replace("_", ""), 16)
```